crates/light-client-prover/src/input.rs

```diff
@@ -9,4 +9,6 @@ pub struct LightClientCircuitInput<Da: DaSpec> {
     pub da_block_header: Da::BlockHeader,
 
     pub batch_prover_da_pub_key: Vec<u8>,
+    pub batch_prover_method_id: Vec<u8>,
+    pub batch_prover_journal: Option<Vec<u8>>,
```

Comment:
at this point this would be called batch_proof_journal

are you sure we have to make the journal an input? maybe since we add assumption it can be verified and extracted from the asumption? just want to make sure.

Reply:
to verify the receipt we need the journal